app/scripts/templates/index.mustache

```diff
@@ -0,0 +1,34 @@
+<div id="main-content" class="card">
```

Comment:
My initial reaction to both `index.js` and `index.mustache` was the same. That that name usually respresents the code directory. Then I saw it's had that name from before, and figured I'd just leave my reactions to myself.